OpenRA.Mods.Common/ColorValidator.cs

```diff
@@ -94,7 +94,7 @@ public Color RandomPresetColor(MersenneTwister random, IEnumerable<Color> terrai
 			if (TeamColorPresets.Any())
 			{
 				Color forbidden;
-				Action<string> ignoreError = _ => { };
+				HashSet<string> ignoreError = new HashSet<string>();
```

Comment:
Two style issues here:
* Prefer `var` over explicit type names
* Variable name no longer matches usage

IMO the best fix for both issues would be to change `IsValid` to include null check before adding to the HashSet and give a default argument value of null. We can then remove the dummy set here completely.